### ISSUE TITLE
ajax and auto-submit

### ISSUE THREAD

**Issue by adamsilver** [adamsilver/f#151]
> Unfortunately, submitting automatically goes against principle 4, to give users control.

H: Hmmm, I don't think so. Typically, one would only fail for WCAG "On Input" if the user's context changes. In this case, the Ajax call is only a side effect. If you mean there's a performance concern for successive Ajax requests where the user is selecting each radio as they move through them — say that! :-)
